contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortRollbackJob.java

```diff
@@ -76,9 +74,7 @@ public void run() throws Exception {
             ExecutorService service = null;
             try {
                 service = Executors.newCachedThreadPool();
-                adminClient = new AdminClient(clusterUrl,
-                                              new AdminClientConfig(),
-                                              new ClientConfig());
+                adminClient = new AdminClient(clusterUrl);
```

Comment:
I searched for the callers of the RollBackJob and there is no reference to it. There is not even a main method which is weird. Will look more into it.